python/lsst/validate/base/metric.py

```diff
@@ -200,26 +233,36 @@ def from_json(cls, json_data):
         metric : `Metric`
             Metric from JSON.
         """
-        specs = [Specification.from_json(v)
-                 for v in json_data['specifications']]
-        params = {k: Datum.from_json(v)
-                  for k, v in json_data['parameters'].items()}
         m = cls(json_data['name'],
                 json_data['description'],
-                json_data['operator_str'],
-                specs=specs,
-                parameters=params,
+                json_data['unit'],
                 reference_doc=json_data['reference']['doc'],
                 reference_page=json_data['reference']['page'],
                 reference_url=json_data['reference']['url'])
         return m
 
-    def __getattr__(self, key):
-        if key in self.parameters:
-            return self.parameters[key]
+    # def __getattr__(self, key):
+    #     if key in self.parameters:
+    #         return self.parameters[key]
+    #     else:
+    #         raise AttributeError("%r object has no attribute %r" %
+    #                              (self.__class__, key))
+
+    def validate(self, quantity):
```

Comment:
Specifications will have a `check` method that tests whether a measurement has passed a specification or so, so there would be nice symmetry with that.